hosted/helpers/helper_common.go

```diff
@@ -76,37 +80,54 @@ func CommonBeforeSuite() Context {
 	config.LoadConfig(rancher.ConfigurationFileKey, rancherConfig)
 
 	testSession := session.NewSession()
-	rancherClient, err := rancher.NewClient(rancherConfig.AdminToken, testSession)
+	rancherAdminClient, err := rancher.NewClient(rancherConfig.AdminToken, testSession)
 	Expect(err).To(BeNil())
 
 	setting := new(management.Setting)
-	resp, err := rancherClient.Management.Setting.ByID("server-url")
+	resp, err := rancherAdminClient.Management.Setting.ByID("server-url")
 	Expect(err).To(BeNil())
 
 	setting.Source = "env"
 	setting.Value = fmt.Sprintf("https://%s", RancherHostname)
-	resp, err = rancherClient.Management.Setting.Update(resp, setting)
+	_, err = rancherAdminClient.Management.Setting.Update(resp, setting)
+	Expect(err).To(BeNil())
+
+	var stduser = namegen.AppendRandomString("stduser-")
+	var stduserpassword = password.GenerateUserPassword("testpass-")
+	newuser := &management.User{
+		Username: stduser,
+		Password: stduserpassword,
+		Name:     stduser,
+		Enabled:  pointer.Bool(true),
+	}
+
+	stdUser, err := users.CreateUserWithRole(rancherAdminClient, newuser, "user")
+	Expect(err).To(BeNil())
+
+	stdUser.Password = newuser.Password
+	stdUserClient, err := rancherAdminClient.AsUser(stdUser)
```

Comment:
Can we initialize this standard user client only in the support matrix suite since that's the only place we will be using it?